packages/inter-protocol/src/stakeFactory/stakeFactoryKit.js

```diff
@@ -71,6 +71,8 @@ const initState = (zcf, startSeat, manager) => {
   const emptyDebt = AmountMath.makeEmpty(debtBrand);
 
   const initialDebt = (() => {
+    // What `makeInvitation` call does this correspond to?
```

Comment:
I think it comes from...

https://github.com/Agoric/agoric-sdk/blob/ef35a91f56d47314f2bed14e7c45c3f319203850/packages/inter-protocol/src/stakeFactory/stakeFactory.js#L201